superset/views.py

```diff
@@ -1801,26 +1801,38 @@ def fave_dashboards(self, user_id):
     @expose("/created_dashboards/<user_id>/", methods=['GET'])
     def created_dashboards(self, user_id):
         Dash = models.Dashboard  # noqa
+        Log = models.Log
         qry = (
             db.session.query(
                 Dash,
+                sqla.func.count(),
+                sqla.func.count(Log.user_id.distinct()),
             )
+            .join(Log)
```

Comment:
`outerjoin`

also note that the query may perform better as a join to a subquery on the Log table that aggregates by dashboard id